automation_testing/tests/form/form_with_layout.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { test, expect } from "@playwright/test";
+import { FormWithLayout } from "../../pages/Form/FormWithLayoutPage";
+
+test.describe("Form - Form With Layout", () => {
+	let formWithLayoutPage: FormWithLayout;
+
+	test.beforeEach(async ({ page }) => {
+		formWithLayoutPage = new FormWithLayout(page);
+		await formWithLayoutPage.visit("form_with_layout");
+	});
+
+	test("Validate that each Section has a Title", async () => {
+		const expectedNumberOfTitles = await formWithLayoutPage.getNumberOfSectionsFromTopComponent();
+		const numberOfSections = await formWithLayoutPage.getNumberOfSections();
+		expect(numberOfSections).toBe(expectedNumberOfTitles);
+		expect(await formWithLayoutPage.getSectionsTitles()).not.toBe("");
+	});
+
+	test("Validate that each Section has a Description", async () => {
+		expect(await formWithLayoutPage.getSectionsDescriptions()).not.toBe("");
+	});
+
+	test("Validate that each row in each section does not have more than 3 columns", async () => {
+		const numberOfColumnsPerRow = await formWithLayoutPage.getNumberOfColumsInEachRowFromSections()
+		for (let i = 0; i < numberOfColumnsPerRow.length; i++) {
+			expect(numberOfColumnsPerRow[i]).toBeLessThanOrEqual(3);
+		}
+	});
+
+	test("Validate that when scrolling to the bottom of the page, the correct Section is selected in Top Component", async () => {
```

Comment:
All tests seem pretty good to me, the only thing I suggest would be to change our titles to be a bit shorter, for example this test could be something like: Correct section selected when scrolling to the bottom of the page.